sdk/textanalytics/azure-ai-textanalytics/pom.xml

```diff
@@ -36,12 +36,22 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.3.0-beta.2</version> <!-- {x-version-update;com.azure:azure-core;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <version>1.4.0-beta.2</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
+    </dependency>
+
+    <!-- Added this dependency to include necessary annotations used by reactor core.
```

Comment:
do other Java libraries have this dependency?

Reply:
Storage has it.